### DISCUSSION TITLE
Switching the QScrollArea default behavior Horizontally

### DISCUSSION THREAD

**Discussion by alitalaghat** [quasarframework/quasar#15704] [General - Components / Directives / etc]
Is there any way to change the default mouse wheel behavior on Scroll Area to Horizontal?

There are some solutions to do this with CSS or JS, but I am not sure how to do these with QScrollArea:
Ref1: https://css-tricks.com/pure-css-horizontal-scrolling/
Ref2: https://stackoverflow.com/a/24639236/5257876

**Comment by pdanpdan** [comment]
Keep shift key pressed. Native

**Reply by alitalaghat** [reply]
I know that, but lots of people don't :D

Also, this can be useful for some creative senario, like this 
https://www.jquerypost.com/demo/horizontal-scroll-one-page-in-jquery-fullpage-js-553/
